.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           node_version: ${{ matrix.node_version }}
 
       - name: E2E tests in chrome
-        run: npm run e2e:chrome && npm run e2e:vue-showcase:chrome
+        run: npx nx affected --target=run e2e:chrome && npx nx affected --target=run e2e:vue-showcase:chrome
```

Comment:
I don't think that will actually work

The idea is, you would run `npx nx affected --target=e2e` and any "projects" (apps or libs) that are considered "affected" by Nx would run the `e2e` target in their respective `project.json` files.

SO running `npx nx affected --target=e2e` for example, looks for anything it thinks has been changed in the repo, it decides that by checking things like git SHAs (passed as  arguments`--base=[SHA1] --head=[SHA2]` usually something like `--base=main --head=HEAD`) or staged files, and then examines the dependency graph to decided what would be affected from those changes.

Then it looks for projects with a listing in `targets` in the `project.json` and runs the commands.

In this case only the `showcase-e2e` and the `vue-showcase` projects have `e2e` targets at all.  So assuming they were **both affected** it would run the:
`@nrwl/cypress:cypress` executor for the `showcase-e2e` project
and the 
`nx-vue3-vite:cypress` executor for the `vue-showcase` project

You can further refine the options passed to the executor by adding `configurations` and adding a `--configuration` argument to the command, so you could limit it to `chrome` or whatever with options there.

The whole idea, is that with all the projects configured correctly, you can just always run the command (or something like it) `yarn nx affected --target=e2e` and only the E2E tests that could've been affected will run, no matter how many more projects you add, the CI script stays the same.

Feel free to ask me questions for clarification